kernel/userspace.c

```diff
@@ -81,7 +81,7 @@ const char *otype_to_str(enum k_objects otype)
 	}
 #else
 	ARG_UNUSED(otype);
-	return NULL;
+	ret = NULL;
```

Comment:
With this change to use variable `ret` instead, this is no longer an early-return #else clause. Is that acceptable?

Reply:
it does return one line later.